src/Components/Publishing/Display/DisplayPanel.tsx

```diff
@@ -87,7 +88,7 @@ export class DisplayPanel extends Component<Props, State> {
 
   @trackImpression(() => "panel")
   trackImpression() {
-    // noop
+    // Add 3rd party impression support
```

Comment:
do we need to use the `track` library for this?  Looks like it isn't implemented either way, but assumed we would need to inject a script tag as well.

Reply:
Na, I thought we did but we actually don't need this her